diff --git a/intake/catalog/local.py b/intake/catalog/local.py
index 262d5f9..bb4b724 100644
--- a/intake/catalog/local.py
+++ b/intake/catalog/local.py
@@ -779,6 +779,10 @@ class YAMLFilesCatalog(Catalog):
                 else:
                     # store a catalog entry
                     self._cats[f.path] = entry
+        import threading
+        print(threading.current_thread().name, "sleeping")
+        import time; time.sleep(3)  # TEST
+        print(threading.current_thread().name, "awake")
         for name, entry in list(self._cats.items()):
             if self._flatten:
                 entry.reload()
